### PR TITLE
[MNOE-1008] Price of product missing

### PR DESCRIPTION
**Pull request by iseessel**

### CODE REVIEW

File: src/app/views/provisioning/confirm.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
           <td>{{ vm.subscription.product_pricing.name }}</td>
           <td>{{ vm.subscription.product_pricing.description }}</td>
           <td>
-            <span ng-hide="PRICING_TYPES['unpriced'].indexOf(pricingPlan.pricing_type) != -1" ng-repeat="price in vm.subscription.product_pricing.prices | filterBy: ['currency']: vm.orgCurrency" class="badge">
+            <span ng-if="vm.pricedPlan(vm.subscription.product_pricing)" ng-repeat="price in vm.subscription.product_pricing.prices | filterBy: ['currency']: vm.orgCurrency" class="badge">
```
Comment (adamaziz15) [inline src/app/views/provisioning/confirm.html]: I'd use `ng-show` here instead as that is what's used throughout this file.

File: src/app/views/provisioning/order.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
           <td>{{ pricingPlan.name }}</td>
           <td>{{ pricingPlan.description }}</td>
           <td>
-            <span ng-hide="PRICING_TYPES['unpriced'].indexOf(pricingPlan.pricing_type) == -1" ng-repeat="price in pricingPlan.prices | filterBy: ['currency']: vm.orgCurrency" class="badge">
+            <span ng-if="vm.pricedPlan(pricingPlan)" ng-repeat="price in pricingPlan.prices | filterBy: ['currency']: vm.orgCurrency" class="badge">
```
Comment (adamaziz15) [inline src/app/views/provisioning/order.html]: `ng-show`

File: src/app/views/provisioning/confirm.html (hunk 1 above)
Comment (adamaziz15) [reply src/app/views/provisioning/confirm.html]: Also I'd only use `ng-if` if for some reason we need the DOM element to be completely removed from the page. `ng-hide` & `ng-show` are usually going to be sufficient (they just affect styling).

Comment (iseessel) [reply src/app/views/provisioning/confirm.html]: @adamaziz15 I am modelling after https://github.com/maestrano/mno-enterprise-angular/pull/433/files -- should I also change these to ng-hide ?

Comment (adamaziz15) [reply src/app/views/provisioning/confirm.html]: @iseessel Yes please, in general I think `ng-hide`/`ng-show` should be used together. It's a bit safer as well as there could be implications to using `ng-if` - it not only removes elements from the DOM, it also creates a child scope.

**Pull request merged**